mock/py/mockbuild/util.py

```diff
@@ -1251,7 +1251,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     config_opts['rpm_command'] = '/bin/rpm'
     config_opts['rpmbuild_command'] = '/usr/bin/rpmbuild'
     config_opts['dnf_disable_plugins'] = ['local', 'spacewalk']
-
+    config_opts['user_agent'] = "Mock ({{ root }}; {{ target_arch }})"
```

Comment:
Strictly speaking, I think we can drop the `target_arch` part as well since it is part of `root`?